notifications.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_sns_topic" "sns-cloudwatch-partinfra" {
+  name = "victorOps-partinfra-topic"
+  display_name = "VictorOps Partinfra"
+}
+
+resource "aws_sns_topic_subscription" "sns-cloudwatch-target" {
```

Comment:
Add `endpoint_auto_confirms = true`. Normally this would be done automatically but the terraform default is `false` which will not work with https